frontend/apps/core-app/src/components/AuthWrapper/useTokenResponse.ts

```diff
@@ -131,7 +135,11 @@ export const useTokenResponse = (): [
       if (refreshTokenInterval.current)
         clearInterval(refreshTokenInterval.current);
     };
-  }, [tokenResponse?.refreshToken, tokenResponse?.expiresIn]);
+  }, [
+    tokenResponse?.refreshToken,
+    tokenResponse?.expiresIn,
+    JSON.stringify(discovery),
```

Comment:
That might be expensive to stringify for change detection. Would this work with `discovery` alone ?

Reply:
Ran a test for it and it's below a ms